exercises/nth-prime/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "nth_prime.h"
+#include <stdbool.h>
+#include <math.h>
+
+static bool is_prime(uint32_t n)
+{
+  for(uint32_t i = (uint32_t)sqrt(n); i > 1; i--) {
```

Comment:
This is a pretty inefficient way to do trial division. Low numbers factor into more numbers than higher numbers, so it's more efficient to start from the bottom. If you wrote it as `for(uint32_t i = 2; i * i < n: ++i)` you could get rid of the `math.h` include as well.

(Sorry for being pedantic on the algorithm, [I've done a lot of prime stuff lately](https://github.com/patricksjackson/primes-cpp))...